docs/addition-tips.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 6
+---
+
+
+# Additional Tips and Tricks
+
+**Boundaries:**
+
+The Meta Quest 3 may prompt the user through a series of brief steps to create a boundary or enable via passthrough. It is recommended to create a boundary \> stationary boundary for use at an office desk.
+
+**Trouble seeing the Vitessce Desktop UI through passthrough?**
+
+Enhance font sizes by moving your head closer to the screen, or increasing your browser size on Mac Command+ and on Windows STRG \+. You can reset the size after the session by using  on Mac Command+0 and on Windows STRG+0
```

Comment:
```suggestion
Enhance font sizes by moving your head closer to the screen, or increasing your browser size on Mac <kbd>Command</kbd> + <kbd>+</kbd> and on Windows <kbd>Control</kbd> + <kbd>+</kbd>. You can reset the size after the session by using  on Mac <kbd>Command</kbd>+<kbd>0</kbd> and on Windows <kbd>Control</kbd>+<kbd>0</kbd>.
```

1. STRG appears to be the German "Control" key, so we should update it to match the fully spelled out "Command" key/the language of the rest of the docs.
2. `<kbd>` is useful for specifying keyboard shortcuts since it also helps avoid ambiguity. For example, for the "Control key + Plus key" shortcut, writing it out as `Command +` is unclear and looks incomplete; I personally assumed there's a missing second key and that the + is just the connector between the two, until I noticed that the `+` was also used for the `Command + 0` case, at which point I was able to make the connection. With `<kbd>`, we avoid the ambiguity and apply some default styles that look familiar to most users:

![image](https://github.com/user-attachments/assets/23329fc0-0a22-4e6f-bbc3-3eccc3954b73)